config/initializers/feature_toggles.rb

```diff
@@ -3,11 +3,7 @@
 beta_testers = ENV['BETA_TESTERS']&.split(/[\s,]+/) || []
```

Comment:
You keep this line just for the record?

Reply:
I would remove it in the same commit. So if we needed to revert this, it's done in one step.